lib/omniauth/strategies/facebooknewflow.rb

```diff
@@ -0,0 +1,4 @@
+class Facebooknewflow < OmniAuth::Strategies::Facebook
+  option :path_prefix, '/i/auth'
+  option :name, 'facebooknewflow'
+end
```

Comment:
Hmm are this one and the Googlenewflow even necessary?

Reply:
Yes, they are. It was the easiest and cleanest way to do it—add the ability to log in with facebook/google in the new flow and remain in the new flow, and have the old flow working at the same time.